nekoyume/Assets/_Scripts/UI/Widget/Menu.cs

```diff
@@ -186,6 +193,8 @@ var firstOpenMimisbrunnrKey
             mimisbrunnrExclamationMark.gameObject.SetActive(
                 btnMimisbrunnr.IsUnlocked
                 && PlayerPrefs.GetInt(firstOpenMimisbrunnrKey, 0) == 0);
+            
+            stakingLevelIcon.sprite = SpriteHelper.GetStakingIcon(States.Instance.StakingLevel);
```

Comment:
It is good too, however we have `MonsterCollectionStateSubject`.
This class has reactive-stream about `StakingLevel`. You can subscribe this stream like it...
```cs
MonsterCollectionStateSubject.Level.Subscribe(level => 
    stakingLevelIcon.sprite = SpriteHelper.GetStakingIcon(level))
    .AddTo(gameObject);
```

Reply:
This is awesome... Thank you!